src/System.Private.CoreLib/shared/System/Activator.RuntimeType.cs

```diff
@@ -141,6 +141,7 @@ public static T CreateInstance<T>()
             return (T)((RuntimeType)typeof(T)).CreateInstanceDefaultCtor(publicOnly: true, skipCheckThis: true, fillCache: true, wrapExceptions: true);
         }
 
+        [System.Runtime.CompilerServices.Intrinsic]
```

Comment:
This should not be marked as Intrinsic. Instead, `CreateInstance<T>()` should be marked as Intrinsic, I think.